src/main-client.ts

```diff
@@ -1219,234 +1890,1383 @@ export class MainClient extends BaseRestClient {
+    return this.postPrivate('sapi/v2/loan/flexible/repay', params);
+  }
+
+  getLoanFlexibleRepaymentHistory(
+    params: GetLoanRepaymentHistoryParams,
+  ): Promise<{
+    rows: LoanRepaymentHistory[];
+    total: number;
+  }> {
+    return this.getPrivate('sapi/v2/loan/flexible/repay/history', params);
+  }
+
+  adjustCryptoLoanFlexibleLTV(
+    params: AdjustFlexibleCryptoLoanLTVParams,
+  ): Promise<AdjustFlexibleCryptoLoanLTVResponse> {
+    return this.postPrivate('sapi/v2/loan/flexible/adjust/ltv', params);
+  }
+
+  getCryptoLoanFlexibleLTVAdjustmentHistory(
+    params: GetFlexibleLoanLTVAdjustmentHistoryParams,
+  ): Promise<{
+    rows: FlexibleLoanLTVAdjustmentHistory[];
+    total: number;
+  }> {
     return this.getPrivate(
-      `sapi/v1/futures/loan/adjustCollateral/history`,
+      'sapi/v2/loan/flexible/ltv/adjustment/history',
       params,
     );
   }
 
-  getCrossCollateralLiquidationHistory(
-    params?: GetLoanCoinPaginatedHistoryParams,
-  ): Promise<any> {
-    return this.getPrivate(`sapi/v1/futures/loan/liquidationHistory`, params);
+  getCryptoLoanFlexibleAssets(params: { loanCoin?: string }): Promise<{
+    rows: FlexibleLoanAssetData[];
+    total: number;
+  }> {
+    return this.getPrivate('sapi/v2/loan/flexible/loanable/data', params);
   }
 
-  getCrossCollateralInterestHistory(
-    params?: GetLoanCoinPaginatedHistoryParams,
-  ): Promise<any> {
-    return this.getPrivate(`sapi/v1/futures/loan/interestHistory`, params);
+  getCryptoLoanFlexibleCollateralAssets(params: {
+    collateralCoin?: string;
+  }): Promise<{
+    rows: FlexibleLoanCollateralAssetData[];
+    total: number;
+  }> {
+    return this.getPrivate('sapi/v2/loan/flexible/collateral/data', params);
+  }
+
+  /**
+   *
+   * Copy trading endpoints
+   *
+   **/
+
+  getFuturesLeadTraderStatus(): Promise<GetFuturesLeadTraderStatusResponse> {
+    return this.getPrivate('sapi/v1/copyTrading/futures/userStatus');
   }
 
-  //TODO: https://binance-docs.github.io/apidocs/spot/en/#futures
+  getFuturesLeadTradingSymbolWhitelist(): Promise<
+    GetFuturesLeadTradingSymbolWhitelistResponse[]
+  > {
+    return this.getPrivate('sapi/v1/copyTrading/futures/leadSymbol');
+  }
 
   /**
    *
-   * BLVT Endpoints
+   * Pay endpoints
    *
    **/
 
-  //TODO: https://binance-docs.github.io/apidocs/spot/en/#blvt-endpoints
+  getPayTransactions(params: GetPayTradeHistoryParams): Promise<any> {
+    return this.getPrivate('sapi/v1/pay/transactions', params);
+  }
 
   /**
    *
-   * BSwap Endpoints
+   * Convert endpoints
+   *
+   **/
+
+  getConvertPairs(params: GetAllConvertPairsParams): Promise<any> {
+    return this.getPrivate('sapi/v1/convert/exchangeInfo', params);
+  }
+
+  getConvertAssetInfo(): Promise<any> {
+    return this.getPrivate('sapi/v1/convert/assetInfo');
+  }
+
+  submitConvertQuoteRequest(params: ConvertQuoteRequestParams): Promise<any> {
+    return this.postPrivate('sapi/v1/convert/getQuote', params);
+  }
+
```

Comment:
Yeah looks like you've renamed the method by accident. This is part of why it's better to split tidying/refactoring into separate PRs from feature work. Reduces the risk of something like this from happening by accident. Please revert the function name back to what it was and check for any other possible breaking changes - need to be careful about renaming any classes/methods/interfaces that have already been released.

Reply:
okay, reverted back to how it was. Accident!